Lib9c/TableCSV/StakeActionPointCoefficientSheet.csv

```diff
@@ -0,0 +1,6 @@
+level,required_gold,coefficient
```

Comment:
How about containing calculated cost of each action rather than calculating cost in action?